scalameta/contrib/shared/src/main/scala/scala/meta/contrib/CommentOps.scala

```diff
@@ -1,46 +1,32 @@
 package scala.meta.contrib
 
-import scala.annotation.tailrec
 import scala.meta.tokens.Token.Comment
 
 object CommentOps {
 
   private[this] val scaladocSymbols: Char => Boolean = Set('*', ' ')
-  private[this] val scaladocBorderSymbols: Char => Boolean = Set('/', '*', ' ')
 
-  @tailrec
-  private def dropRightWhile(str: String, predicate: (Char) => Boolean): String =
-    if (str.isEmpty)
-      ""
-    else if (predicate(str.last))
-      dropRightWhile(str.init, predicate)
-    else
-      str
+  @inline
```

Comment:
Let's drop `@inline` annotations even if they were there before, we have no indication these help with performance bottlenecks

Reply:
will do